sponge/src/main/java/me/lucko/luckperms/sponge/service/model/ContextCalculatorProxy.java

```diff
@@ -30,67 +30,48 @@
 import net.luckperms.api.context.ContextConsumer;
 
 import org.checkerframework.checker.nullness.qual.NonNull;
+import org.spongepowered.api.event.Cause;
 import org.spongepowered.api.service.context.Context;
 import org.spongepowered.api.service.context.ContextCalculator;
 import org.spongepowered.api.service.permission.Subject;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.function.Consumer;
 
 public class ContextCalculatorProxy implements ForwardingContextCalculator<Subject> {
-    private final ContextCalculator<Subject> delegate;
+    private final ContextCalculator delegate;
 
-    public ContextCalculatorProxy(ContextCalculator<Subject> delegate) {
+    public ContextCalculatorProxy(ContextCalculator delegate) {
         this.delegate = delegate;
     }
 
     @Override
     public void calculate(@NonNull Subject subject, @NonNull ContextConsumer consumer) {
-        this.delegate.accumulateContexts(subject, new ForwardingContextSet(consumer));
+        throw new UnsupportedOperationException();
```

Comment:
This is getting thrown for me when I do /lp user <name> info
Not sure if this is intentional as its still in progress but I thought i'd point it out since I noticed it.

Reply:
Thanks :)